front/src/interfaces/molecules/ChannelSettingItem/IChannelSettingItem.ts

```diff
@@ -0,0 +1,12 @@
+import { UserType } from '../../utils/api/IUser';
+
+export interface ChannelSettingItemProps {
+    user: UserType;
+    isAdmin: boolean;
+    isOwner: boolean;
+    showUserProfile: () => void;
+    kickOutUser: () => void;
+    muteUser: () => void;
+    setAdmin: () => void;
+    unSetAdmin: () => void;
```

Comment:
```suggestion
    unsetAdmin: () => void;
```
unset도 하나의 단어입니다. 카멜케이스로 네이밍 하면 위와 같이 수정하는게 좋을 것 같아요